mock-thunderhead/src/main/resources/application.yml

```diff
@@ -33,3 +33,4 @@ auth:
     differentdatahubversionthandatalake.enabled: true
     database.wire.encryption.enable: true
     datahub.runtime.upgrade.enable: true
+    embedded.database.on.attached.disk.enable: true
```

Comment:
My recommendation is to make it false by default. We'll enable it once we have a reserved disk for it.

Reply:
even for the mock?

Reply:
mock means everyone who is developing CB. If you say it's fine we can try with true (at least people are going to test it without knowing it :) )

Reply:
Default value has set to false